tests/system/conftest.py

```diff
@@ -78,18 +90,37 @@ def table_id(dataset_id):
     return f"{dataset_id}.table_{helpers.temp_suffix()}"
 
 
-@pytest.fixture(scope="session")
-def scalars_table(bigquery_client: bigquery.Client, project_id: str, dataset_id: str):
+def load_scalars_table(
+    bigquery_client: bigquery.Client,
+    project_id: str,
+    dataset_id: str,
+    data_path: str = "scalars.jsonl",
+) -> str:
     schema = bigquery_client.schema_from_json(DATA_DIR / "scalars_schema.json")
     job_config = bigquery.LoadJobConfig()
     job_config.schema = schema
     job_config.source_format = enums.SourceFormat.NEWLINE_DELIMITED_JSON
     full_table_id = f"{project_id}.{dataset_id}.scalars"
```

Comment:
Need random/custom table ID to prevent conflicts with "extremes" and regular table.